src/cc2olx/main.py

```diff
@@ -27,13 +27,18 @@ def convert_one_file(input_file, workspace, link_file=None):
 
     tgz_filename = (workspace / cartridge.directory.name).with_suffix(".tar.gz")
 
-    filesystem.add_in_tar_gz(
-        str(tgz_filename),
-        [
-            (str(olx_filename), "course.xml"),
-            (str(cartridge.directory / "web_resources"), "/static/"),
-        ],
-    )
+    file_list = [
+        (str(olx_filename), "course.xml"),
+        (str(cartridge.directory / "web_resources"), "/static/"),
+    ]
+
+    # Add static files that are outside of web_resources directory
+    file_list += [
+        (str(cartridge.directory / filepath), "/static/{}".format(filepath))
```

Comment:
Nit: We can make use of  `OLX_STATIC_DIR` here just to keep it consistent throughout. what do you say?

Reply:
Yes, I agree, thanks :-)